packages/gateway/index.ts

```diff
@@ -1,54 +1,20 @@
-import { ApolloServer, gql } from 'apollo-server';
+import { ApolloServer } from 'apollo-server';
 import { ApolloGateway } from '@apollo/gateway';
-import * as console from 'console';
+import * as fs from 'fs';
+import * as path from 'path';
 
-gql`
-    scalar _Any
-    scalar _FieldSet
-
-    # a union of all types that use the @key directive
-    union _Entity = Weather | City
-
-    type _Service {
-        sdl: String
-    }
-
-    type Query {
-        _entities(representations: [_Any!]!): [_Entity]!
-        _service: _Service!
-    }
-
-    directive @external on FIELD_DEFINITION
-    directive @requires(fields: _FieldSet!) on FIELD_DEFINITION
-    directive @provides(fields: _FieldSet!) on FIELD_DEFINITION
-    directive @key(fields: _FieldSet!) repeatable on OBJECT | INTERFACE
-
-    # this is an optional directive discussed below
-    directive @extends on OBJECT | INTERFACE
-`;
+const supergraphSdl = fs
+    .readFileSync(path.resolve(__dirname, './supergraph.graphql'))
+    .toString();
 
 const gateway = new ApolloGateway({
-    serviceList: [
-        {
-            name: 'city',
-            url: 'http://localhost:4001',
-        },
-        {
-            name: 'weather',
-            url: 'http://localhost:4002',
-        },
-    ],
+    supergraphSdl,
 });
 
-(async () => {
-    const { schema, executor } = await gateway.load();
-
-    const server = new ApolloServer({
-        schema,
-        executor,
-    });
+const server = new ApolloServer({
+    gateway,
```

Comment:
I ended up adding the following to this so I could play around in the browser locally.
```suggestion
    gateway,
    plugins: [ApolloServerPluginLandingPageGraphQLPlayground()],
```

Reply:
Ah cool, you can use the graphql playground locally which is probably a bit nicer than the browser

https://github.com/graphql/graphql-playground